src/Datastore/DatastoreClient.php

```diff
@@ -0,0 +1,796 @@
+    {
+        $this->operation->mutate('delete', $keys, Key::class);
+        return $this->operation->commit($options);
+    }
+
+    /**
+     * Retrieve an entity from the datastore
+     *
+     * Example:
+     * ```
+     * $key = $datastore->key('person', 'Bob');
+     *
+     * $entity = $datastore->lookup($key);
+     * if (!is_null($entity)) {
+     *     echo $entity['firstName']; // 'Bob'
+     * }
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @param Key $key $key The identifier to use to locate a desired entity
+     * @param array $options {
+     *     Configuration Options
+     *
+     *     @type string $readConsistency See
+     *           [ReadConsistency](https://cloud.google.com/datastore/reference/rest/v1beta3/ReadOptions#ReadConsistency).
+     *     @type string $className The name of the class to return results as.
+     *           Must be a subclass of {@see Google\Cloud\Datastore\Entity}.
+     *           If not set, {@see Google\Cloud\Datastore\Entity} will be used.
+     * }
+     * @return Entity|null
+     * @codingStandardsIgnoreEnd
+     */
+    public function lookup(Key $key, array $options = [])
+    {
+        $res = $this->lookupBatch([$key], $options);
+
+        return (isset($res['found'][0]))
+            ? $res['found'][0]
+            : null;
+    }
+
+    /**
+     * Get multiple entities
+     *
+     * Example:
+     * ```
+     * $keys = [
+     *     $datastore->key('Person', 'Bob'),
+     *     $datastore->key('Person', 'John')
+     * ];
+     *
+     * $entities = $datastore->lookup($keys);
+     *
+     * foreach ($entities['found'] as $entity) {
+     *     echo $entity['firstName'];
+     * }
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @param Key[] $key The identifiers to look up.
+     * @param array $options {
+     *     Configuration Options
+     *
+     *     @type string $readConsistency See
+     *           [ReadConsistency](https://cloud.google.com/datastore/reference/rest/v1beta3/ReadOptions#ReadConsistency).
+     *     @type string $className The name of the class to return results as.
+     *           Must be a subclass of {@see Google\Cloud\Datastore\Entity}.
+     *           If not set, {@see Google\Cloud\Datastore\Entity} will be used.
+     * }
+     * @return array
+     * @codingStandardsIgnoreEnd
+     */
+    public function lookupBatch(array $keys, array $options = [])
+    {
+        return $this->operation->lookup($keys, $options);
+    }
+
+    /**
+     * Create a Query
+     *
+     * The Query class can be used as a builder, or it can accept a query
+     * representation at instantiation.
+     *
+     * Example:
+     * ```
+     * $query = $datastore->query();
+     * ```
+     *
+     * @param array $options {
+     *     Query Options
+     *
+     *     @type array $query [Query](https://cloud.google.com/datastore/reference/rest/v1beta3/projects/runQuery#query)
```

Comment:
```php
// wow, such query!
$query = $datastore->query([
    'query' => [
    // options
    ]
]);
```

This seems to be a little easier to type imo, is there a reason we wouldn't be able to do this?
```php
$query = $datastore->query([
    // options
]);